packages/frontend-app-api/src/wiring/parameters.ts

```diff
@@ -251,8 +289,33 @@ export function mergeExtensionParameters(options: {
     })),
   ];
 
+  // Install all extension overrides
+  for (const extension of overrideExtensions) {
+    // Check if our override is overriding an extension that already exists
+    const index = configuredExtensions.findIndex(
+      e => e.extension.id === extension.id,
+    );
+    if (index !== -1) {
+      // Only implementation, attachment point and default disabled status are overridden, the source is kept
+      configuredExtensions[index].extension = extension;
+      configuredExtensions[index].params.attachTo = extension.attachTo;
+      configuredExtensions[index].params.disabled = extension.disabled;
+    } else {
```

Comment:
Just a  conceptual question:
Suppose we have a `SearchResultItem` extension and we would like to attach it to multiple parents, would that be correct to create the extension once and provide multiple overrides for attaching it to different parents (I mean, specifying different ids and attachment points)?

Reply:
The integrator has to do this because plugins cannot provide overrides, right 🤔 ?

Reply:
Makes sense, thanks for the clarifications 🙂